examples/gno.land/r/leon/home/home.gno

```diff
@@ -31,6 +33,8 @@ My contributions to gno.land can mainly be found
 TODO import r/gh
 `,
 	}
+
+	_, _ = hof.RegisterFor(seqid.ID(1))
```

Comment:
multiple exhibitions; plenty returned values that are never used

please, try to make minimalist contracts; that's what we need at the moment

Reply:
Removed temporary exhibitions, just have one now.